pkg/point/client.go

```diff
@@ -0,0 +1,118 @@
+package point
+
+import (
+	"context"
+
+	"github.com/mercadopago/sdk-go/pkg/config"
+	"github.com/mercadopago/sdk-go/pkg/internal/baseclient"
+)
+
+const (
+	urlBase                = "https://api.mercadopago.com/point"
+	urlDevices             = urlBase + "/integration-api/devices/"
+	urlPaymentIntent       = urlDevices + ":device_id/payment-intents"
+	urlPaymentIntentSearch = urlBase + "/integration-api/payment-intents/:payment_intent_id"
+	urlPaymentIntentCancel = urlDevices + ":device_id/payment-intents/:payment_intent_id"
+	urlDevicesWithID       = urlDevices + ":device_id"
+)
+
+// client is the implementation of Client.
+type client struct {
+	cfg *config.Config
+}
+
+// Client contains the methods to interact with the Point API.
+type Client interface {
+
+	// Create a point payment intent.
+	// It is a post request to the endpoint: https://api.mercadopago.com/point/integration-api/devices/{device_id}/payment-intents
+	// Reference: https://www.mercadopago.com/developers/en/reference/integrations_api_paymentintent_mlb/_point_integration-api_devices_deviceid_payment-intents/post
+	Create(ctx context.Context, deviceID string, request Request) (*CreateResponse, error)
+
+	// Get a point payment intent.
+	// It is a search request to the endpoint: https://api.mercadopago.com/point/integration-api/payment-intents/{payment_intent_id}
+	// Reference: https://www.mercadopago.com/developers/en/reference/integrations_api/_point_integration-api_payment-intents_paymentintentid/get
+	Get(ctx context.Context, paymentIntentID string) (*SearchResponse, error)
+
+	// Cancel a point payment intent.
+	// It is a cancel request to the endpoint: https://api.mercadopago.com/point/integration-api/devices/{device_id}/payment-intents/{payment_intent_id}
+	// Reference: https://www.mercadopago.com/developers/en/reference/integrations_api/_point_integration-api_devices_deviceid_payment-intents_paymentintentid/delete
+	Cancel(ctx context.Context, deviceID string, paymentIntentID string) (*CancelResponse, error)
+
+	// GetDevices search devices.
+	// It is a search request to the endpoint: https://api.mercadopago.com/point/integration-api/devices
+	// Reference: https://www.mercadopago.com/developers/en/reference/integrations_api/_point_integration-api_devices/get
+	GetDevices(ctx context.Context) (*DevicesResponse, error)
+
+	// UpdateDeviceOperationMode getintentstatus device.
+	// It is a patch request to the endpoint: https://api.mercadopago.com/point/integration-api/devices/{device-id}
+	// Reference: https://www.mercadopago.com/developers/en/reference/integrations_api/_point_integration-api_devices_device-id/patch
+	UpdateDeviceOperationMode(ctx context.Context, deviceID string, request UpdateDeviceOperatingModeRequest) (*OperationModeResponse, error)
```

Comment:
se só tem um campo na request não podíamos fazer como fazemos para o cancel de payments?
além disso, mudar o nome do método
```suggestion
	UpdateDeviceOperatingMode(ctx context.Context, deviceID, operatingMode string) (*OperationModeResponse, error)
```

Reply:
Alterado!!